packages/migrate/src/commands/MigrateDiff.ts

```diff
@@ -26,6 +28,7 @@ const helpOptions = format(
 ${bold('Options')}
 
   -h, --help               Display this help message
+  -o, --output             Path to the file where the diff output will be written to
```

Comment:
Small inspiration from `curl -o`'s help message.

```suggestion
  -o, --output             Writes to a file instead of stdout
```